src/main/java/com/systemmeltdown/robotlib/subsystems/drive/SingleSpeedTalonDriveSubsystem.java

```diff
@@ -0,0 +1,76 @@
+package com.systemmeltdown.robotlib.subsystems.drive;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.FeedbackDevice;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.systemmeltdown.robotlib.commands.DriveProportionalCommand;
+
+public class SingleSpeedTalonDriveSubsystem extends SkidSteerDriveSubsystem {
+    private WPI_TalonSRX m_rightMaster;
+    private WPI_TalonSRX m_leftMaster;
+    private WPI_TalonSRX[] m_rightSlaves;
+    private WPI_TalonSRX[] m_leftSlaves;
+
+    public SingleSpeedTalonDriveSubsystem(int rightMasterId, int leftMasterId, int[] rightSlaveIds,
+            int[] leftSlaveIds) {
+
```

Comment:
The configuration in this constructor looks fine for a first run, but I feel this code should be in an overridden `configure` method, and the first line in the configure should be to set the talons back to factory configuration.

See here for an example: https://github.com/CrossTheRoadElec/Phoenix-Examples-Languages/blob/master/Java/ArcadeDrive_ArbFeedForward/src/main/java/frc/robot/Robot.java